doc/source/whatsnew/v1.1.0.rst

```diff
@@ -144,7 +144,7 @@ Reshaping
 - Bug in :meth:`DataFrame.pivot_table` when only MultiIndexed columns is set (:issue:`17038`)
 - Fix incorrect error message in :meth:`DataFrame.pivot` when ``columns`` is set to ``None``. (:issue:`30924`)
 - Bug in :func:`crosstab` when inputs are two Series and have tuple names, the output will keep dummy MultiIndex as columns. (:issue:`18321`)
-
+- Bug in :func:`concat` index and columns not copied when ``copy=True`` (:issue:`29879`)
```

Comment:
Bug in ....concat, where the resulting indices where not copied when ``copy=True`` (the default).

Reply:
updated